src/components/Missions.jsx

```diff
@@ -9,29 +11,29 @@ const Missions = () => {
   useEffect(() => {
     dispatch(fetchMissions());
   }, [dispatch]);
-
+  console.log(missions); // eslint-disable-line no-console
   return (
```

Comment:
- [ ] Kindly remove `console.log` statement in your code, because it's one of the best practices to follow in React.